tests/internet.bats

```diff
@@ -6,7 +6,17 @@ load test-helper
   assert_success && assert_output -p 'true'
 }
 
+@test "$clinom internet (invalid argument)" {
+  run "${clicmd}" internet foobar
+  assert_failure && assert_output -p 'incorrect command'
+}
+
 @test "$clinom internet reverse" {
   run "${clicmd}" internet reverse
   assert_success
```

Comment:
This could also assert partial output for the 6 categories of `internet reverse`.